static/js/hotkey.js

```diff
@@ -301,6 +301,14 @@ exports.process_enter_key = function (e) {
         return true;
     }
 
+    if (popovers.reactions_popped()) {
+        var selected_emoji = $(".reaction-popover-reaction").filter(":focus")[0];
```

Comment:
I would do one of the following things here to move the above jquery code into reaction.js:
* Make a little helper called `reactions.get_selected_emoji`
* Put that logic into `toggle_reaction` (and then maybe rename it `maybe_toggle_reaction`).

I assume even if you don't toggle the reaction here, we want to return `true`, correct?  That seems correct to me, just checking.